drs_filer/app_config.yaml

```diff
@@ -0,0 +1,50 @@
+# General server/service settings
```

Comment:
I think now with FOCA and a single config file, I'd prefer to just put the file in the root directory of the repo and call it `config.yaml`

Reply:
Please just name it `config.yaml`, as there is only one config file left now.